### ISSUE TITLE
Extend the Religion Quest to include denomination

### ISSUE THREAD

**Issue by peternewman** [westnordost/StreetComplete#1737]
**Use case**
Some denominations are clearly marked on religious buildings (e.g. Jehovahs Witness, Salvation Army), some people may either want to add the extra detail, or may not be clear on exactly which parent religion they are part of.

**Proposed Solution**
Using the data already here, which groups by religion and denomination:
https://wiki.openstreetmap.org/wiki/Key:denomination?uselang=en-GB

Either filter out "popular"/easily identifiable ones, or have a keyboard search list. Styles like the surface one, so paved>concrete becomes Christian>Jehovahs Witness, I can select either Christian, and it just populates religion, or Jehovahs Witness and it populates both religion AND denomination.

**Comment by westnordost** [comment]
Going through that list to select which denominations are actually important enough so that they should be shown in the app (= translated into every language, an icon drawn for it) will be quite some effort. I expect that certain denominations will also be somewhat local.

Given a look at the statistics #1749 and the fact that `AddReligionToPlaceOfWorship` was only solved 794 times so far, I am doubtful if it would be worth the effort.

**Comment by matkoniecz** [comment]
> AddReligionToPlaceOfWorship was only solved 794 times so far, I am doubtful if it would be worth the effort.

Maybe new quest asking about denomination would make more sense (I thought about it and even planned it, but decided to skip this as `denomination` tag is unused or barely used).

**Comment by peternewman** [comment]
> Going through that list to select which denominations are actually important enough so that they should be shown in the app (= translated into every language, an icon drawn for it) will be quite some effort. I expect that certain denominations will also be somewhat local.

I get the feeling far more work went into the orchard quest for example, for even fewer answers. I don't think they need icons necessarily (indeed I'm not sure how you'd necessarily iconify some while keeping them clear. I was essentially thinking something like the building type or surface quest, but the sub level could just be a list of denomination names. Given the huge list on the wiki, I wonder if just suggesting the popular ones (possibly per area/country) could work, but perhaps this would be deemed discriminatory?

> > AddReligionToPlaceOfWorship was only solved 794 times so far, I am doubtful if it would be worth the effort.
> 
> Maybe new quest asking about denomination would make more sense (I thought about it and even planned it, but decided to skip this as `denomination` tag is unused or barely used).
> 
> There is plenty of `amenity=place_of_worship` without `denomination` and religion value where asking about `denomination` would add useful info.

My reason for suggesting they were combined is that it means you don't have to know the religion a certain denomination is attached to (and given they are inherently linked, you don't have to answer two quests, you just pick the most detailed answer you can, the same as building type), and it populates the fields as necessary.

**Comment by matkoniecz** [comment]
> I get the feeling far more work went into the orchard quest for example, for even fewer answers.

Yes, but orchard quest has the worst ratio of usage to development effort.

I can try implementing limited version just for Christianity - here I am relatively certain about my knowledge and probably I can implement this without terribly offending anyone and without glaring errors. For other religions I am not going to implement it because I lack necessary background knowledge.

**Comment by peternewman** [comment]
> > I get the feeling far more work went into the orchard quest for example, for even fewer answers.
> 
> Yes, but orchard quest has the worst ratio of usage to development effort.

I think I've seen the orchard quest on a map three times, and attempted to answer it twice. Once I was trying to guess from the tree labels and had to just leave a note. I think I then found an info-board which confirmed they were apples. The other time I couldn't see anything being grown in the orchard area. The third time was too far off my route so I didn't visit it.

> I can try implementing limited version just for Christianity - here I am relatively certain about my knowledge and probably I can implement this without terribly offending anyone and without glaring errors. For other religions I am not going to implement it because I lack necessary background knowledge.

Thanks @matkoniecz . The wiki page seems to be fairly clear on choices and names, or do you mean regarding popularity/likelihood etc?

**Comment by matkoniecz** [comment]
In general I think it can be implemented rather than as a separate quest as

1) in many cases religion is tagged and denomination is missing
2) at least some people will have no knowledge/interest in tagging this detail

> Thanks @matkoniecz . The wiki page seems to be fairly clear on choices and names, or do you mean regarding popularity/likelihood etc?

I am simply unable to say whatever this wiki page for say https://wiki.openstreetmap.org/wiki/Key:denomination?uselang=en-GB#Hindu_denominations is actually valid classification or deeply misleading in some way.
I am also not sure whatever you can actually neatly say "this temple is of Smartism denomination and that one is of Shaktism denomination" like you can generally say with Christian denominations.
I would be unable to even spot blatant jokes (checking Wikipedia may help here).
I would be unable to spot clearly missing data (even for christian denominations my ability will be limited).

**Comment by kmpoppe** [comment]
The list in the Wiki is rather extensive, and I would also be totally lost in spotting errors ("jokes") or missing entries. As much as I'm a religious person myself, I'd hate to see conflicts about what denomination would be suitable to list/the sort order of said list arise for a tool as great as SC and I would therefore lean towards not implementing such a quest. If, however, we do, I'd recommend to deactivate the quest as a standard (just like the speedlimit one), to prevent people from being tasked with a quest they simply cannot answer from plain sight.

**Comment by matkoniecz** [comment]
I posted https://lists.openstreetmap.org/pipermail/tagging/2021-December/063270.html that was triggered by research for this quest

**Comment by matkoniecz** [comment]
Currently blocked on https://github.com/openstreetmap/id-tagging-schema/pull/263
